src/screens/Learn/Products/ProductPage.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment } from 'react';
+import { useParams } from 'react-router-dom';
+import { formatMetatagDate } from 'common/utils';
+import AppMetaTags from 'components/AppMetaTags/AppMetaTags';
+import Breadcrumbs from 'components/Breadcrumbs';
+import { MarkdownContent, Heading1, Heading2 } from 'components/Markdown';
+import PageContent from 'components/PageContent';
+import {
+  learnPages,
+  getProductPageContent,
+  productPageContent,
+} from 'cms-content/learn';
+import { BreadcrumbsContainer } from '../Learn.style';
+
+const ProductPage: React.FC = () => {
+  let { productId } = useParams<{ productId: string }>();
+  const date = formatMetatagDate();
+  const pageContent = getProductPageContent(productId);
+
+  if (!productPageContent) {
+    return null;
+  }
+
+  const {
+    productName,
+    productIntro,
+    metadataTitle,
+    metadataDescription,
+    sections,
+  } = pageContent[0];
```

Comment:
We probably need a map to fetch pageContent given a product ID (like we did for the case studies). We can do this on a follow-up PR to unblock content entry

Reply:
that is already set up. `pageContent` is the return value of `getProductPageContent(productId)` a few lines up, which is the function I put in `learn/index` (image below). i used `.filter` so i just needed to grab the obj from the return array, hence the `[0]`

![Screen Shot 2020-11-09 at 7 36 18 PM](https://user-images.githubusercontent.com/44076375/98612895-26cf0280-22c3-11eb-839e-26220f2f818a.png)

Reply:
oh, how about using `find` instead then?